mvc/views/tag-helpers/authoring/sample/TagHlp/src/AuthoringTagHelpers/TagHelpers/zBoldTagHelperCopy2.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+
+namespace AuthoringTagHelpers.TagHelpers2
+{
+[TargetElement("bold", Attributes = "bold")]
```

Comment:
The source code is driven by the docs. Perhaps next version I'll keep the source consistent and fiddle with :dedent: more.